### DISCUSSION TITLE
is it possible to use `where` and `count` to return the count for a specified field?

### DISCUSSION THREAD

**Discussion by adherb** [prisma/prisma#11280] [Q&A] [answered]
I am trying to use Prisma to return a count for a boolean field where it equals 'true'.

To give some context, on the frontend I am trying to calculate the workouts that have been completed by a user as a percentage, so ideally I would like prisma to return a count for the total workouts (which I have successfully done) and the count for the userWorkouts where 'isCompleted' equal true (which I am unable to achieve), currently the count is returning all userWorkouts not just the completed ones.

Here is my current Prisma Query:

```
const response = await prisma.user.findUnique({
  where: {
    id: 1,
  },
  select: {
    id: true,
    programs: {
      select: {
        program: {
          select: {
            name: true,
            blocks: {
              select: {
                id: true,
                name: true,
                week: {
                  select: {
                    id: true,
                    number: true,
                    workouts: {
                      select: {
                        userWorkouts: {
                          where: {
                            isCompleted: true,
                          },
                        },
                        _count: {
                          select: {
                            userWorkouts: true,
                          },
                        },
                      },
                    },
                    _count: {
                      select: {
                        workouts: true,
                      },
                    },
                  },
                },
              },
            },
          },
        },
      },
    },
  },
});
res.json(response);
};
```

Is this possible to achieve using Primsa? Or should I just return all userWorkouts and filter for isCompleted: true on the frontend?

**Comment by austincrim** [comment, marked as answer]
Hey @adherb 👋 ,

This is not possible yet but we do have an [existing feature request](https://github.com/prisma/prisma/issues/8413) for it. It would be great if you could add your +1 and specific use case to help our engineering teams prioritize!

In the meantime, the workaround you suggested should work fine.

**Reply by adherb** [reply]
Hey @austincrim, awesome thank you, will do.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
